plugins/module_utils/_api_data.py

```diff
@@ -727,15 +727,16 @@ def join_path(path):
             },
         ),
     ),
-    ('ip', 'route', 'vrf'): APIData(
+    ('ip', 'vrf'): APIData(
```

Comment:
This breaks all users of older RouterOS versions. Better add a copy of this path for RouterOS 7 with the new path, and keep the old one for previous versions.